project/locale.py

```diff
@@ -0,0 +1,126 @@
+"""This file is used to store all user facing text,
+where Starwort then transcribes it into katakana."""
+
+
+class kata:
+    class menu:
+        class unhelpful:
```

Comment:
this is a very strange way to use classes, and the class names are not PEP8 compliant. Feels perhaps a little bit too creative to be pythonic, although I see what you were going for.

Reply:
These classes are more akin to enums than classes, hence the incompliance to PEP8 classname rules. Personally I feel this is more pythonic than my old dot accessing hacks. I could have used an empty class and added its attributes that way also, but again, this feels more unpythonic. Hence the solution as it is here.